service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionService.java

```diff
@@ -49,6 +49,7 @@
 public class CaseDataExtractionService {
 
     public static final String EMPTY_PLACEHOLDER = "BLANK - please complete";
+    public static final String HEARING_EMPTY_PLACEHOLDER = "This only appear in the issued CMO";
```

Comment:
explain?

Reply:
As discussed, next hearing date is only going to be present on the following events 

Action CMO (Judge) - About to Submit
Draft CMO (LA) when returned via Judge - Mid event (if exists)
Draft CMO (LA) when returned via Judge - About to submit event (if exists)

Theo and design agreed that "BLANK - please complete" is not appropriate when LA cannot populate the next hearing Date. Because it's not always going to be present on the draft CMO, content decided to alter the empty state message :/